src/packages/@app/pages/Teams/Teams.ViewModel.old.ts

```diff
@@ -0,0 +1,359 @@
+import { WorkspaceService } from "@app/services/workspace.service";
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { WorkspaceRepository } from "@app/repositories/workspace.repository";
+import {
+  requestResponseStore,
+  tabs,
+} from "$lib/store/request-response-section";
+import { TabRepository } from "@app/repositories/tab.repository";
+import { CollectionRepository } from "@app/repositories/collection.repository";
+import { TeamService } from "@app/services/team.service";
+import { TeamRepository } from "@app/repositories/team.repository";
+import type { TeamDocument } from "@app/database/database";
+import type { Observable } from "rxjs";
+import type { InviteBody } from "$lib/utils/dto/team-dto";
+import type { TeamRole, WorkspaceRole } from "$lib/utils/enums";
+import { UserService } from "@app/services/user.service";
+import type { MakeRequestResponse } from "$lib/utils/interfaces/common.interface";
+import type { Team } from "$lib/utils/interfaces";
+
+export class TeamViewModel {
+  constructor() {}
+  private workspaceRepository = new WorkspaceRepository();
+  private tabRepository = new TabRepository();
+  private collectionRepository = new CollectionRepository();
+  private workspaceService = new WorkspaceService();
+  private teamService = new TeamService();
+  private teamRepository = new TeamRepository();
+  private userService = new UserService();
+
+  public debounce = (func, delay) => {
+    let timerId;
+
+    return function (...args) {
+      /* eslint-disable @typescript-eslint/no-this-alias */
+      const context = this;
+
+      clearTimeout(timerId);
+      timerId = setTimeout(() => {
+        func.apply(context, args);
+      }, delay);
+    };
+  };
```

Comment:
this file is not required

Reply:
you have introduced 2 new view model, that is not required. just rename the existing one to old and create only 1 view model.